packages/storybook8/stories/Concepts/Identity.mdx

```diff
@@ -0,0 +1,39 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta
```

Comment:
meta needs to be changed to `<Meta title="Concepts/Identity" />`

Reply:
cleaned it up to just be the meta tag with the title property